Source/WarpX.H

```diff
@@ -779,6 +782,7 @@ public:
     bool getis_synchronized() const {return is_synchronized;}
 
     int maxStep () const {return max_step;}
+    void updateMaxStep (int const new_max_step) {max_step = new_max_step;}
```

Comment:
Are there other parts of the code where `max_step` is overwritten and which could reuse this new function? If so, I would be in favor of reusing this function there, too, since it is now available. For example, we might be doing something similar when we compare `max_step` and `zmax_plasma_to_compute_max_step`, but I'm not 100% sure.

Reply:
The parameter``zmax_plasma_to_compute_max_step``  is used to compute ``max_step`` in a ``WarpX`` member function, so ``max_step`` is directly available.  The use of ``updateMaxStep`` in this function is deep in a Diagnostics initialization function and so perhaps dangerous.  I couldn't think of a safer way to change ``max_step`` or I would avoid the use of this function.